config/routes.rb

```diff
@@ -9,4 +9,8 @@
   resource :sessions, only: [:create, :destroy]
   
   get '/login', to: 'sessions#new'
+
+  resource :keywords, only: :create
+
+  get '/keywords', to: 'keywords#new'
```

Comment:
we could use `resource :keywords, only: %i[newcreate]`, it's normal Rails application

Reply:
If I put `new` in resource then rails will generate which is I didn't want.
```ruby
new_keywords     GET     /keywords/new(.:format)     keywords#new
```

What I want is just only `/keywords`. Do you have any ideas?

Reply:
Normally the `/keywords` is used for `index page` @gutakk.

Here, you can use the `/keywords/new` for now, till you have the index page front-end (which display all user's keywords, and you can add the HTML form on that page) then you can remove the `/keywords/new` route and using only `/keywords`

Reply:
Fixed in b9e86049cb46a6b6121a7f19de36808b9262e9f3